src/components/task/TaskCard.jsx

```diff
@@ -0,0 +1,13 @@
+import { FaEdit, FaTrashAlt } from "react-icons/fa";
+
+export function TaskCard({ title }) {
+  return (
+    <section className="task-card flex justify-between align-center">
+      <p className="task-title">{title}</p>
+      <div className="mr-1">
+        <FaEdit style={{ marginRight: "1rem", cursor: "pointer" }} />
```

Comment:
utility classes can be used instead of inline CSS?

Reply:
Changes made.